multinet.rst

```diff
@@ -1,12 +1,122 @@
+for the clubs they belong to. Multinet recognizes nodes encoded in different
+tables as being of different node types, so download :download:`clubs.csv
+</files/clubs.csv>` and upload it to a ``clubs`` table, using the same procedure
+as above.
+
+You should now see both a ``members`` and a ``clubs`` table listed. You can
+click on each of these to see the rows of each table. Use the browser's back
+button to return to the workspace view.
+
+**3. Upload an Edge Table**
+
+Next we want to specify the topology of this network---that is, the relationship
+between clubs and members as specified by membership. We need a special table
+that encodes these memberships, using special attributes ``_from`` and ``_to``.
+These attributes will always refer to rows from other tables in the workspace,
+using a *node ID*, which is formed by joining the name of the table and the key
+of the row with a slash. For instance, the John Adams row has a key value of
+``7``; its node ID is therefore ``members/7``.
+
+You can think of the rows of such a table as a directed links connecting two of
+the nodes from somewhere in the workspace. To see this in action, download
+:download:`/files/membership.csv`, and upload it as a table. Because this table
+has properly formatted ``_from`` and ``_to`` attributes, Multinet recognizes it
+as an *edge table*. If you click on the newly uploaded table, you can explore
+what the rows look like. Note that while edge tables can also have other
+attributes, this one does not.
+
+**4. Create a Network**
+
+Creating a network in Multinet is a matter of specifying an edge table that
+defines that network's topology and implicitly (via the ``_from`` and ``_to``
+values of its edge table rows, the nodes involved as) its nodes as well. To
+create the Paul Revere network, click on the plus icon to the right of the
+Networks column. When the dialog box appears, click on the ``CREATE`` tab at the
+top, then select the ``membership`` table from the dropdown. Note that Multinet
+already knows which tables are edge tables, due to the presence of ``_from`` and
+``_to`` attributes. Fill in the name of the network with ``boston``, and then
+click on the ``CREATE NETWORK`` button. You should see the new network appear in
+the Networks column.
+
+If you click on the new network, you will be taken to the network view, which
+aggregates various pieces of information about this network, and provides links
+to visualization applications. Note that the main panel displays some
+characeristic information such as the number of nodes and edges, and the drawer
+at the bottom displays information about the node types comprising this network,
+the edge table that provides the connectivity, and a paged view of the nodes
+themselves, identified by their node IDs. Clicking on these items will bring you
+to dedicated views providing specific information; you can return to the network
+view using the browser back button.
```

Comment:
This feels clunky, do we not have UI for this?

Reply:
I'm not sure, but it's worth teaching the user that the routing in this app works as though they're just "on the web".